### PR TITLE
Deprecate Xpanse Behavior commands

### PR DESCRIPTION
**Pull request by dorschw**
Implementing https://github.com/demisto/content/pull/16218

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: see aforementioned contribution PR 

## Description
deprecating `expanse-get-risky-flows` and `expanse-list-risk-rules`, as they're no longer supported on the Xpanse API.
Calling one of these commands will show an error message.

## Screenshots
image

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [x] Yes
       - Further details: deprecated commands, errors will show.
   - [ ] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/ExpanseV2/ReleaseNotes/1_9_5.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Cortex Xpanse
+- **Breaking Changes:** The `expanse-get-risky-flows` and `expanse-list-risk-rules` commands are no longer supported in the Xpanse API, and have been deprecated.
```
Comment (dantavori) [inline Packs/ExpanseV2/ReleaseNotes/1_9_5.md:4]:
```suggestion
- **Breaking Changes:** The ***expanse-get-risky-flows*** and ***expanse-list-risk-rules*** commands are no longer supported in the Xpanse API, and have been deprecated.
```

Comment (dantavori) [reply Packs/ExpanseV2/ReleaseNotes/1_9_5.md:4]: let's use the [`-bc` flag](https://xsoar.pan.dev/docs/documentation/release-notes#breaking-changes-version)

File: Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.py [hunk 2]
```diff
@@ -123,7 +123,7 @@ def __init__(
         hdr = {
             "Content-Type": "application/json",
             "Accept": "application/json",
-            "User-Agent": "Expanse_XSOAR/1.7.0",
+            "User-Agent": "Expanse_XSOAR/1.10.0",
```
Comment (dantavori) [inline Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.py:126]: should this match the pack version number?

**Review by dantavori — APPROVED** [review]

**Comment by YuvHayun** [comment]
@dorschw In this PR, you deprecated the following commands
'expanse-list-risk-rules': which is being used in 'Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-SolarStorm_and_SUNBURST_Hunting_and_Response_Playbook_6.yml'.
And
'expanse-get-risky-flows' which is being used in 'Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-SolarStorm_and_SUNBURST_Hunting_and_Response_Playbook_6.yml'

Can you take a look at that and try to replace the mentioned commands in those places?

**Comment by dorschw** [comment]
> @dorschw In this PR, you deprecated the following commands 'expanse-list-risk-rules': which is being used in 'Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-SolarStorm_and_SUNBURST_Hunting_and_Response_Playbook_6.yml'. And 'expanse-get-risky-flows' which is being used in 'Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-SolarStorm_and_SUNBURST_Hunting_and_Response_Playbook_6.yml'
> 
> Can you take a look at that and try to replace the mentioned commands in those places?

Please open a Jira issue for that

**Pull request merged**